server/monitor/pom.xml

```diff
@@ -36,6 +36,11 @@
       <artifactId>auto-service</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-core</artifactId>
+      <version>2.3.0.1</version>
```

Comment:
The version should be specified in the the root `pom.xml`'s `<dependencyManagement />` section instead of here, and the dependency here should be listed as `<scope>runtime</scope>`

(Fixing the scope, may fix the build. I'm not sure.)